tests/console/commands/test_init.py

```diff
@@ -242,6 +248,24 @@ def test_interactive_with_git_dependencies(
     assert expected in tester.io.fetch_output()
 
 
+@pytest.mark.parametrize("package_name", ["flask", "Flask", "flAsK"])
+def test_generate_choice_list(tester: CommandTester, package_name: str):
```

Comment:
Can you add another test, that shows that if there are more than 10 packages, the list is truncated and the package we are searching is still the first result?

Reply:
@finswimmer I've added the truncation condition to the test you highlighted. Felt that extending the package list to > 10 items, then checking for both conditions which result from the `_generate_choice_list()` method would be cleaner. If you'd like me to break them out for separation of concerns, I can also do that - just wanted to keep it DRY.